docs/ENV.md

```diff
@@ -66,3 +66,4 @@ The following environment variables are available to configure the bom generatio
 | SWIFT_COMPILER_ARGS             | Full compiler arguments string to use for semantic analysis. Eg: -sdk <path> -F <path> -Xcc -I <path>                                                                                                                                                                                                                               |
 | SWIFT_SDK_ARGS                  | Swift sdk arguments. Eg: -sdk <path>                                                                                                                                                                                                                                                                                                |
 | SWIFT_COMPILER_EXTRA_ARGS       | Extra compiler arguments to add to the auto-detected string. Eg: -suppress-warnings -track-system-dependencies                                                                                                                                                                                                                      |
+| GO_FETCH_VCS                   | Set this variable to `true` or `1` to fetch vsc url from pkg.go.dev. golang                                                                                                                                                                                                                                   |
```

Comment:
Typo. Change vsc to vcs. Also should anything come before the word golang such as "for golang"?

Reply:
Fixed